backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/CassandraCluster.java

```diff
@@ -129,6 +122,7 @@ public CassandraTypesProvider getTypesProvider() {
 
     @Override
     public void close() {
-        cluster.closeAsync();
+        FutureConverter.toCompletableFuture(session.closeAsync())
```

Comment:
AFAIR the session closing is useless, it should be called when calling cluster.closeAsync.

Reply:
Ok, I'll revert that part of the change.